examples/SampleApp/src/components/NewDirectMessagingSendButton.tsx

```diff
@@ -4,49 +4,30 @@ import { TouchableOpacity } from 'react-native-gesture-handler';
 import { useNavigation } from '@react-navigation/core';
 
 import {
-  DefaultAttachmentType,
-  DefaultChannelType,
-  DefaultCommandType,
-  DefaultEventType,
-  DefaultMessageType,
-  DefaultReactionType,
-  DefaultUserType,
+  DefaultStreamChatGenerics,
   MessageInputContextValue,
   Search,
   SendRight,
   SendUp,
-  UnknownType,
   useChannelContext,
   useMessageInputContext,
   useTheme,
 } from 'stream-chat-react-native';
 
 import { NewDirectMessagingScreenNavigationProp } from '../screens/NewDirectMessagingScreen';
 
-import { StreamChatGenerics } from '../types';
+import { StreamChatGenerics as LocalStreamChatGenerics } from '../types';
 
 type NewDirectMessagingSendButtonPropsWithContext<
-  At extends UnknownType = DefaultAttachmentType,
-  Ch extends UnknownType = DefaultChannelType,
-  Co extends string = DefaultCommandType,
-  Ev extends UnknownType = DefaultEventType,
-  Me extends UnknownType = DefaultMessageType,
-  Re extends UnknownType = DefaultReactionType,
-  Us extends UnknownType = DefaultUserType,
-> = Pick<MessageInputContextValue<At, Ch, Co, Ev, Me, Re, Us>, 'giphyActive' | 'sendMessage'> & {
+  StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics,
```

Comment:
Cleaned out some remnants of the ancient times before `StreamChatGenerics` were introduced.